CalcualtorLogic.js

```diff
@@ -103,9 +99,14 @@ class Calculator {
     onButtonNumberClick(event) {
         const currentResultAndclickedButton = this.currentResult + event.target.dataset.number;
         this.currentResult = currentResultAndclickedButton;
+        console.log(currentResultAndclickedButton)
+    }
+    updateResultElement() {
+        const updatedResult = document.getElementById('result').innerHTML = this.currentResult;
```

Comment:
Assigning the `this.currentResult` is correct.
but getting the resultElement is incorrect.

I was wondering if you want to use `document.getElementById`  to select the resultElement then why did you save it as property on the class in the render method?

search in the render method for this line 
`  this.resultElement = resultDiv;` 
this.resultElement is the result element.
you don't need to use document.getElementById or document.querySelector 
this element is in front of you and you don't need to search for it

you can think of it as the returned value of calling `document.getElementById`
in other words, what `document.getElementById` returns?
it returns a HTMl elements
`resultElement` is also a HTML elements but this time you have created that element in the render method 
so you don't need to search for it, it's in front of you and ready to be used